### DISCUSSION TITLE
How can I access the 'tags' field of an mp4 media file in windows 10 21H2?

### DISCUSSION THREAD

**Discussion by TEDSON007** [mono/taglib-sharp#272] [Q&A]
In windows 10 file explorer the field is accessed via its details tab in properties window.

!! Question title has been edited because of a mistake and assumption on my part.

![tagsproperty](https://user-images.githubusercontent.com/36574091/136704901-66517af6-d334-48c2-850a-dfeb50001c82.jpg)

**Comment by Vasilich** [comment]
I don't have "Tags" field in Windows explorer for MP3 - only for MP4. W10 21H2.

**Reply by TEDSON007** [reply]
You're correct of course. My apologies.

I'll edit question so to not waste anyone else's time.

Thank you.

That is what I'm after.

**Comment by Panzerbjrn** [comment]
I just came to look for this as well. When I try to add a tag to the property, I'm told "InvalidOperation: 'Tags' is a ReadOnly property." -__-
